ksql-engine/src/main/java/io/confluent/ksql/util/SerDeUtil.java

```diff
@@ -67,24 +69,28 @@
     return Serdes.serdeFrom(genericRowSerializer, genericRowDeserializer);
   }
 
-  public static Serde<GenericRow> getGenericRowAvroSerde(final Schema schema) {
+  public static Serde<GenericRow> getGenericRowAvroSerde(final Schema schema, final KsqlConfig ksqlConfig) {
```

Comment:
This relates more to the calling method. Rather than having an `if(serde instanceof blah)..` etc it would be much nicer if there was a method on `KsqlTopicSerDe` that you could delegate to get the serde.

Reply:
Good point. Done.